### PR TITLE
Clean up docs for abs() and accumulate_n()

### PR DESCRIPTION
**Pull request by frreiss**
This PR adds some clarifications to the documentation for `tf.abs()` and reformats the docs for `tf.accumulate_n()` into a format more consistent with that of `tf.add_n()`.

### CODE REVIEW

**Review by olicht — COMMENTED** [review]
Why did you delete lines 2173-2175 (accumulate_n)?

**Comment by frreiss** [comment]
@olicht I moved the information about the `shape` and `tensor_dtype` arguments to the "Arguments" section at lines 2198-2201 of `math_ops.py`. I can add that information back into the main body of the comment if you'd like.

BTW, any idea why the CI tests don't seem to be running on this PR?

File: tensorflow/python/ops/math_ops.py [hunk 1]
```diff
@@ -130,8 +134,8 @@ def abs(x, name=None):  # pylint: disable=redefined-builtin
     name: A name for the operation (optional).
 
   Returns:
-    A `Tensor` or `SparseTensor` the same size and type as `x` with absolute
-      values.
+    A `Tensor` or `SparseTensor` the same size, type, and sparsity as `x` with 
+      absolute values.
```
Comment (yifeif) [inline tensorflow/python/ops/math_ops.py]: nit, remove space

File: tensorflow/python/ops/math_ops.py [hunk 2]
```diff
@@ -2170,13 +2174,10 @@ def add_n(inputs, name=None):
 def accumulate_n(inputs, shape=None, tensor_dtype=None, name=None):
   """Returns the element-wise sum of a list of tensors.
 
-  Optionally, pass `shape` and `tensor_dtype` for shape and type checking,
-  otherwise, these are inferred.
-
```
Comment (yifeif) [inline tensorflow/python/ops/math_ops.py]: Maybe we can keep this?

**Review by yifeif — CHANGES_REQUESTED** [review]
Thanks @frreiss. Just a few nits.

**Comment by yifeif** [comment]
Thanks @frreiss! Only user with write access can trigger CI. I can help triggering them after the comments have been addressed.

File: tensorflow/python/ops/math_ops.py (hunk 1 above)
Comment (frreiss) [reply tensorflow/python/ops/math_ops.py]: Removed, thanks for catching that. Note to self: Run CI tests on documentation branches :)

File: tensorflow/python/ops/math_ops.py (hunk 2 above)
Comment (frreiss) [reply tensorflow/python/ops/math_ops.py]: Added this back in, along with an additional note that `shape` will override incomplete shape information on the output of the op, which can affect type inference for downstream ops.

**Comment by frreiss** [comment]
Pushed some additional changes to address review comments.

**Comment by tensorflowbutler** [comment]
Nagging Assignee @yifeif: It has been 14 days with no activity and this issue has an assignee. Please update the label and/or status accordingly.

**Comment by frreiss** [comment]
I'm going to fold this PR into a larger documentation PR I've been working on.

**Pull request closed without merging**